### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Saira-A**
Issue #1203 - just the governance doc for now but will add more as we get them.
Also put the 'scope' section back as it had been commented out but the anchor was still there.

### CODE REVIEW

**Comment by LanieOkorodudu** [comment]
@Saira-A This looks good to me. Let’s wait for @Demian’s feedback on the README update to see what he thinks.

**Comment by Saira-A** [comment]
> @Saira-A and @LanieOkorodudu, I like the governance link you've added!
> 
> I see that you've also uncommented the Scope section. I'm not sure why that was commented out in the first place (the Git history is unfortunately not enlightening). I honestly don't really like the "Scope" heading -- not sure what it really means; I would be more inclined to call this "About" or "Project History" if it needs a heading. But more than that, I wonder if we should delete the heading entirely and just move the "about the project" link to some other section. I don't think it's the first thing most people are going to want to read about, so having it take up so much real estate so early in the document seems unhelpful.

Thanks! I agree - I just put it back since the anchor link for it was still there but leading to nothing. Since the about section is part of the Wiki I could just link to that instead, perhaps at the top next to the Slack/Twitter links?

**Comment by demiankatz** [comment]
Sounds like a good plan, @Saira-A -- I agree that it makes sense to just remove the scope link from the nav menu (which was probably only left there as an oversight) and add an about link at the top where you suggest.

File: README.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ A community-developed open source project on a mission<br/> to help you share yo
 <img src="https://api.netlify.com/api/v1/badges/91dc58e8-49dd-495f-98bb-84570a0edb7c/deploy-status" />
 </a>
 <a href="https://github.com/UniversalViewer/universalviewer/blob/main/LICENSE.txt"><img src="https://camo.githubusercontent.com/e80e20b31b4af7da8580f68d415779d250eee229/68747470733a2f2f696d672e736869656c64732e696f2f6e706d2f6c2f74687265652e737667" alt="License" data-canonical-src="https://img.shields.io/npm/l/universalviewer.svg" style="max-width:100%;"></a>
+<a href="https://github.com/UniversalViewer/universalviewer/wiki"> About</a>
```
Comment (demiankatz) [inline README.md:15]: I'm going to remove the extra space here:

```suggestion
<a href="https://github.com/UniversalViewer/universalviewer/wiki">About</a>
```

(Just using the "suggestion" feature of GitHub to document what I'm doing).

**Review by demiankatz — APPROVED** [review]
Looks good to me now. Thanks again!

**Comment by demiankatz** [comment]
(We can merge at the meeting in 30 minutes if nobody else objects to anything).

**Pull request merged**